packages/origin-ui-core/src/components/Organization/Organization.tsx

```diff
@@ -85,10 +82,9 @@ export function Organization() {
             key: 'register-irec',
             label: 'Register I-REC',
             component: IRECRegisterForm,
-            show: enabledFeatures.includes(OriginFeature.IRec) && organization && !irecAccount
+            show: organization && iRecAccount.length === 0
```

Comment:
Will this still be hidden if we disable the I-REC feature?

I see that you removed `enabledFeatures.includes(OriginFeature.IRec)`

Reply:
Yes, this feature now depends on existing organization and existing irecAccount. 
IRecConnect is hidden with enabledFeatures.

Reply:
So what you're saying is that putting `enabledFeatures.includes(OriginFeature. IRecConnect)` here would be redundant because of `iRecAccount.length === 0`.

If we are sure the length will always be 0 when we have  `OriginFeature. IRecConnect` disabled, then I guess it's fine.

Reply:
Wait, this is the I-REC registration form?

We want to have this under a feature toggle, because we have Origin instances where we don't use I-REC.

Reply:
Ok, will be done this way.